### ISSUE TITLE
Ability to use a react component as a header row column name

### ISSUE THREAD

**Issue by zpfled** [adazzle/react-data-grid#277]
Hi, I would love to be able to pass in a React node as a column name so that I can, for example, indicate required fields with a red asterisk in the header row. 

<img width="359" alt="screen shot 2016-04-11 at 9 18 44 am" src="https://cloud.githubusercontent.com/assets/5231407/14430352/d6bfd34c-ffc6-11e5-9b8e-4c54d4815af5.png">

Doing this works, but generates a bunch of React 'Failed Proptype' warnings, like so:

```
Warning: Failed propType: Invalid prop `column.name` of type `object` supplied to `HeaderCell`, expected `string`. Check the render method of `HeaderRow`.
```

I got rid of most of them by creating a custom RowRenderer, but I still get the one for the Header Row.

**Comment by jpdriver** [comment]
hey @zpfled -- there's a PR to change those PropTypes to enable that here https://github.com/adazzle/react-data-grid/pull/276. seems quite a few people want it! :smile:

**Comment by bakesteve** [comment]
closed by #276 
grab master and you should be good 
our CI server will bump and publish a new version, so you shoudl be good to grab that via npm once [this](https://ci.appveyor.com/project/adazzle/react-data-grid/build/1.0.1174) is done

**Comment by zpfled** [comment]
Oh, great, thanks @jpdriver
